radixdlt/src/main/java/com/radixdlt/mempool/MempoolReceiver.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * (C) Copyright 2020 Radix DLT Ltd
+ *
+ * Radix DLT Ltd licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied.  See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
+
+package com.radixdlt.mempool;
+
+import java.util.Objects;
+
+import org.radix.logging.Logger;
+import org.radix.logging.Logging;
+
+import com.google.inject.Inject;
+import com.radixdlt.common.Atom;
+import com.radixdlt.network.MempoolNetworkRx;
+
+import io.reactivex.rxjava3.disposables.Disposable;
+
+/**
+ * Network glue for SubmissionControl.
+ */
+public final class MempoolReceiver {
```

Comment:
Could this be the receiver from both mempool and the api?

Reply:
And so all submissions can be contained in a single backpressured flow?

Reply:
Yeah, could be.  Let me have a look.

Reply:
Hmm.  Need to discuss.  Will ping you tomorrow.